### PR TITLE
Adding support to read and write MO files with msgctxt

### PR DESCRIPTION
**Pull request by neohunter**

### CODE REVIEW

File: lib/get_pomo/mo_file.rb [hunk 1]
```diff
@@ -24,6 +25,11 @@ def add_translations_from_text(text)
       text = StringIO.new(text)
       @translations += GetPomo::GetText::MOFile.open(text, "UTF-8").map do |msgid,msgstr|
         translation = Translation.new
+
+        if has_context? msgid
+          translation.msgctxt, msgid = msgid.split CONTEXT_SEPARATOR
+        end
```
Comment (grosser) [inline lib/get_pomo/mo_file.rb]: using the same logic in 2 places always smells like future trouble to me ... prefer
```
translation.msgctxt, msgid = msgid.split CONTEXT_SEPARATOR, 2 if msgid.include?(CONTEXT_SEPARATOR)
```

File: lib/get_pomo/version.rb [hunk 2]
```diff
@@ -1,3 +1,3 @@
 module GetPomo
-  VERSION = "0.9.2"
+  VERSION = "0.9.3"
```
Comment (grosser) [inline lib/get_pomo/version.rb]: undo that plz or version bump will fail ... should generally not do that in PRs

Comment (grosser) [reply lib/get_pomo/version.rb]: that's what broke the travis build too ... test PR worked fine: https://github.com/grosser/get_pomo/pull/24

**Comment by grosser** [comment]
sweet and simple PR with tests ❤️ just needs some tiny cleanup :)

File: lib/get_pomo/version.rb (hunk 2 above)
Comment (neohunter) [reply lib/get_pomo/version.rb]: Sorry, can you explain this? so the version bump is automatically? something increase automatically the version number? can you please explain me what does that?

**Comment by grosser** [comment]
Usually the project maintainer increases the version and not a pull request
...
just change it back and travis will pass and I can merge this.

Version is bumped with `rake bump:patch` after merge

**Comment by neohunter** [comment]
I updated MR, please check and let me know if it's okey now.

**Comment by grosser** [comment]
1.0.0 ... 0.10 looks silly ... so here it goes :)

**Pull request merged**